docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,54 @@
+# Pull Request Template
+
+## Description
+
+:question: Please include a summary of the change(s) a small paragraph in your own words is fine.
+
+
+:question: Break Down of Task, steps taken.
+
+-[ ] ...
+
+-[ ] ...
+
+-[ ] ...
+
+-[ ] ...
+
+-[ ] ...
+
+:question: Have you used any tutorials or done any reading that helped you to complete these changes and would be of benefit to the group? If yes you can add links here. Great for our growth but also so we can help if needed.
```

Comment:
Can we please have a comma after changes, another after yes, and another after growth?